plugins/runner-ts/__tests__/default.test.ts

```diff
@@ -85,3 +85,31 @@ test('Transpiles TypeScript', async () => {
 
   expect(result).toMatchSnapshot();
 });
+
+
+test('Transpiles TypeScript when configFile contains npm module in extends field', async () => {
```

Comment:
I was unable to get it done without using a custom jest resolver.   My resolver is based on the one mentioned in that issue except it's scoped so that module-resolution files for different tests don't affect each other.  It's maybe a bit much for one test, but maybe it can be useful to have.  What do you think?